docs/modules/ROOT/pages/advanced-how-tos/managing-security-fix.adoc

```diff
@@ -1 +1,100 @@
-= Managing a security fix
\ No newline at end of file
+= Managing a security fix
+
+Sometimes, you need to address an **embargoed security issue** in your application. So as not to disclose aspects of the security vulnerability prematurely, you will want to prepare a fix in private or with a limited set of collaborators. This document guides you on how to make a private copy of your Git repository and clone your application to a private workspace where you can prepare a fix and invite other collaborators only on a need-to-know basis, so that you can control who is aware of the issue while you develop the fix.
+
+Key steps include:
+
+* **Clone affected application:** Cloning affected applications into a private workspace, enabling developers to tackle security issues without exposing sensitive information.
+* **Prepare the fix:** Preparing the fix with source code in your private git mirror and builds in your private workspace.
+* **Release the fix:** Releasing the build from your private workspace and "upstreaming" your patch from your private git mirror back to your team's normal development repo.
+
+== Cloning applications to a private workspace
+Cloning an application involves replicating the application's code and configuration into a dedicated private workspace, ensuring that sensitive information remains isolated and protected.
+
+.**Prerequisites**
+
+* You have the Maintainer role or greater in the original workspace where the embargoed issue was identified.
+// Maintainer role was xref:getting-started/roles_permissions/ Currently no such file in konflux-ci/docs. Add a link if the doc is published.
+* You have link:https://konflux-ci.dev/docs/getting-started/cli/[enabled CLI access for {ProductName}].
+
+.**Procedure**
+
+. Create a link:https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository[Private Mirror] of the corresponding repository for any components that will receive fixes related to embargoed information. This step is crucial to safeguard sensitive data and prevent unauthorized access.
+
++
+NOTE: Creating a private fork of the public link:https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository[GitHub repository] is not possible.
+
+. link:https://konflux-ci.dev/docs/getting-started/#for-developers[Request a new workspace] from your platform engineer. This workspace will serve as the private environment for cloning the affected application. Avoid adding information about the vulnerability to a workspace name.
+
++
+[IMPORTANT]
+====
+* If you already have a private workspace that you used for another embargoed issue, you can re-use it. Always double-check the link:https://console.redhat.com/preview/application-pipeline/access[User Access] permissions.
```

Comment:
Would you say the verb "double-check" is direct enough? The gdoc said "remember to review".

Reply:
Can we explicitly state something like to ensure that the minimal set of required individuals have access to the workspace?

Reply:
Sure. I've reworded. Does it work better?

* If you already have a private workspace that you used for another embargoed issue, you can re-use it. Remember to check the [User Access](https://console.redhat.com/preview/application-pipeline/access) permissions and ensure that access to the workspace is granted to a minimal number of required collaborators.

* You can reuse a personal workspace, but limit the [User Access](https://console.redhat.com/preview/application-pipeline/access) to a minimal number of required collaborators.

The only thing: the AppStudio guide linked to https://console.redhat.com/preview/application-pipeline/access which I bet won't work for Konflux users. Do you think we could link to the architecture page https://github.com/konflux-ci/architecture/blob/main/ADR/0011-roles-and-permissions.md ? Until we have a separate guide (Managing access to a workspace) published. Or I can remove the links altogether. Thank you.